oci.go

```diff
@@ -32,6 +32,7 @@ const (
 	cgroupsDirMode   = os.FileMode(0750)
 	cgroupsFileMode  = os.FileMode(0640)
 	cgroupsMountType = "cgroup"
+	cgroupFsType     = 0x27e0eb
```

Comment:
Feels like if there is no go lib that proves that constant, then at least a comment to explain the magic is in order :-)